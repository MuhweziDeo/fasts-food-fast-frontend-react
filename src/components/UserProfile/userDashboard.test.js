import React from 'react';
import UserDashboard from './userDashboard';
import {shallow} from 'enzyme';

describe('<UserDashboard/>',() => {
    it('should render without crushing', () => {
        const props ={
            orderList:[{}],
            menu:[{}]
        }
        let wrapper = shallow(<UserDashboard {...props}/>)
        expect(wrapper).toMatchSnapshot()
    })
})
