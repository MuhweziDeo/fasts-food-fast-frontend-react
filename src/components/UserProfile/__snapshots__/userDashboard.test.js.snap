// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserDashboard/> should render without crushing 1`] = `ShallowWrapper {}`;
